app/models/tipline_newsletter.rb

```diff
@@ -0,0 +1,247 @@
+class TiplineNewsletter < ApplicationRecord
+  SCHEDULE_DAYS = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday']
+
+  include TiplineNewsletterImage
+  include TiplineNewsletterVideo
+  include TiplineNewsletterAudio
+
+  belongs_to :team
+  has_many :tipline_newsletter_deliveries
+  has_paper_trail on: [:create, :destroy], save_changes: true, ignore: [:updated_at, :created_at], versions: { class_name: 'Version' }
```

Comment:
You can add `if: proc { |_x| User.current.present? }`  condition to paper trail to make sure version created with `whodunnit` user

Reply:
Yes, good idea - it also made me realize that I need to track changes for the `:update` event as well.